src/install/unix.rst

```diff
@@ -230,9 +234,55 @@ CouchDB no longer ships with any daemonization scripts.
 The couchdb team recommends `runit <http://smarden.org/runit/>`_ to
 run CouchDB persistently and reliably. Configuration of runit is
 straightforward; if you have questions, reach out to the CouchDB
-user mailing list.
+user mailing list or IRC-channel #couchdb in FreeNode network.
+
+Let's consider configuring of runit on Ubuntu 16.04. The following
+steps should be considered only as an example. Some details can vary
+on different systems and versions of Ubuntu Linux.
+
+Install ruinit::
+
+    sudo apt-get install runit
+   
+Create a directory where logs will be written::
+
+    sudo mkdir /var/log/couchdb
+    sudo chown couchdb:couchdb /var/log/couchdb
+   
+Create directories that will contain runit counfiguration for CouchDB::
+
+    sudo mkdir /etc/sv/couchdb
+    sudo mkdir /etc/sv/couchdb/log
+   
+Create /etc/sv/couchdb/log/run script::
+
+    #!/bin/sh
+    exec svlogd -tt /var/log/couchdb
```

Comment:
it might be worth explaining briefly what these commands actually do, especially if you're not a runit user.